rasterio/plot.py

```diff
@@ -50,6 +50,9 @@ def show(source, cmap='gray', with_bounds=True):
         rather than pixel coordinates. Only works when source is
         (raster dataset, bidx).
     """
+    if plt is None:  # pragma: no cover
+        raise ImportError("Could not import matplotlib")
+
```

Comment:
@ColinTalbert I made a cut and paste error above: I meant that the code of https://github.com/ColinTalbert/rasterio/blob/faf10ed7c04ee6410e92b0d545a05b1598c11617/rasterio/plot.py#L17-L28 could become `get_plt()`. Sorry for the confusion!